pages/views/examples/animate.njk

```diff
@@ -9,43 +9,43 @@ iconAlt: '[esl-animate]'
 <section>
   <h2>Grouped and repetitive animation</h2>
   <ul class="animate-ul">
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
-    <li class="animate-ul-group animate-ul-li" esl-animate></li>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
+    <esl-animate class="animate-ul-group animate-ul-li get-upstairs" esl-animate repeat group="100"></esl-animate>
```

Comment:
Incorrect html we should replace `ul` tag then
Only `li` tags allowed for the `ul`